packages/sdk/src/vc-tools.mjs

```diff
@@ -19,6 +38,82 @@ const staticVC = {
   credentialSubject: {},
 };
 
+/**
+ * Fetches the keys from the server, requires an authenticated session.
```

Comment:
Anyone should be able to query a public key from the did registry ?  not sure why authentication is required ? 

Error retrieving claim JSON: HTTPError: Response code 401 (Unauthorized)
Claim cannot be verified